### DISCUSSION TITLE
Version 14.3 enable flash

### DISCUSSION THREAD

**Discussion by Sickman987** [sanni/cartreader#1017] [Need help]
Am I able to dump the rom of repo cartridges if I enable flash.
If so, i must be doing something wrong. I tried to dump my repro of doki doki panic. With no success. I tried it by selecting nes/famicom and was greeted with a list of games to select from. Then I tried it by selecting flash but all I was able to get was a VLC file.  

Can somebody help me out. I’m still very new to ROM dumping and arduino sketch.

**Comment by sanni** [comment]
I don't think dumping Doki Doki Panic is supported by the OSCR. 

According to nesdev it is mapper 103:
https://forums.nesdev.org/viewtopic.php?t=18956
https://www.nesdev.org/wiki/INES_Mapper_103

And mapper 103 has not been added to the OSCR yet:
https://github.com/sanni/cartreader/blob/master/Cart_Reader/NES.ino#L38

To answer your initial question: For NES the ENABLE_FLASH option allows you to write Infinite Nes Lives NESmaker PCBs and Aliexpress A29040B repros.

**Reply by Sickman987** [reply]
Thank you for your help
